backend/requirements/base.txt

```diff
@@ -1,8 +1,8 @@
 django==4.2.11
-psycopg2-binary
-django-tenants
-djangorestframework
-django-rest-knox
+psycopg2-binary==2.9.9
+django-tenants==3.6.1
+djangorestframework==3.15.2
+django-rest-knox==5.0.1
```

Comment:
@deepakdinesh1123 I checked in our docker images, for `django-rest-knox` version is `4.2.0`. Can you pls change this?